src/lib/openjp2/openjpeg.h

```diff
@@ -667,6 +667,8 @@ typedef struct opj_image_comp {
  * Defines image data and characteristics
  * */
 typedef struct opj_image {
+    /** RSIZ: capabilities */
+    OPJ_UINT16 rsiz;
```

Comment:
to avoid ABI issues this should be moved as the last component of the structure

Reply:
Oh right thanks, did not trigger an issue in my CI build weird.